### PR TITLE
Increase page size to prevent api throttle error

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝

This PR sets the value of `page_size` to either `25` or `500` based on the current value of `window.location.pathname`. This change helps avoid the API from throttling users who switch routes frequently resulting in a 429 error code from the server.

## Preview 📷
**`page_size=25`**

page-size-25

**`page_size=500`**

page-size-500

## How to test 🧪
Log into Cloud Manager with a user account that has more than one StackScript. Next navigate to any other page and then quickly return to the StackScripts landing page. User should NOT experience any 429 server codes.

**How do I run relevant unit or e2e tests?**
`yarn test`

### CODE REVIEW

**Review by bnussman-akamai — CHANGES_REQUESTED** [review]
This seems to fix this issue of hitting the rate limit, but now, rather than the `Community StackScripts` tab loading 25 items at a time, it seems to just load 500. Can we keep the old behavior on the `Community StackScripts` tab?

**Comment by carrillo-erik** [comment]
> This seems to fix this issue of hitting the rate limit, but now, rather than the `Community StackScripts` tab loading 25 items at a time, it seems to just load 500. Can we keep the old behavior on the `Community StackScripts` tab?

I see the side effect this is causing, I will look into it. Thanks for catching it.

**Comment by cpathipa** [comment]
I agree with @bnussman-akamai point, As long as we don't get rate limit issue by page_size to 25 only for community tab we should be good.

**Review by cpathipa — COMMENTED** [review]
@carrillo-erik We can use prop `category` to set page size for community.

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptBase.tsx [hunk 1]
```diff
@@ -146,7 +146,14 @@ const withStackScriptBase = (options: WithStackScriptBaseOptions) => (
         currentPage: page,
       });
 
-      return request({ page, page_size: 25 }, filter)
+      return request(
+        {
+          page,
+          page_size:
+            window.location.pathname === '/stackscripts/community' ? 25 : 500,
```
[the remark this answers is not in the document]
Comment (carrillo-erik) [reply packages/manager/src/features/StackScripts/StackScriptBase/StackScriptBase.tsx]: Thank you for showing me this, I had missed it. I've made the change.

**Review by cpathipa — APPROVED** [review]

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**